README.md

```diff
@@ -22,16 +22,16 @@ bucket for SSL certificates in the COOL DNS account.
 | Name | Version |
 |------|---------|
 | terraform | ~> 0.12.0 |
-| aws | ~> 2.0 |
+| aws | ~> 3.0 |
 
 ## Providers ##
 
 | Name | Version |
 |------|---------|
-| aws | ~> 2.0 |
-| aws.dnsprovisionaccount | ~> 2.0 |
-| aws.organizationsreadonly | ~> 2.0 |
-| terraform | ~> 2.0 |
+| aws | ~> 3.0 |
+| aws.dnsprovisionaccount | ~> 3.0 |
+| aws.organizationsreadonly | ~> 3.0 |
+| terraform | n/a |
```

Comment:
Should this be 3.0 instead of n/a?

Reply:
Never mind, I see it's `n/a` in other repos if it's not included as `aws.terraform`. Since this is an auto-generated addition, I withdraw my comment :)

Reply:
`n/a` is correct in this case- it refers to the Terraform provider itself (which we have no version constraints on), not the Terraform **AWS** provider (which we are now requiring be 3.x).